app/move/controllers/update/base.js

```diff
@@ -0,0 +1,114 @@
+const { get, keys } = require('lodash')
+
+const personService = require('../../../../common/services/person')
+const CreateBaseController = require('../create/base')
+
+class UpdateBaseController extends CreateBaseController {
+  middlewareLocals() {
+    super.middlewareLocals()
+    this.use(this.setStepUrls)
+    this.use(this.setInitialStep)
+  }
+
+  _setModels(req) {
+    const res = req.res
+    req.models.move = res.locals.move
+    req.models.person = req.models.move.person
+  }
+
+  getBaseUrl(req) {
+    const moveId = req.getMoveId()
+    return `/move/${moveId}`
+  }
+
+  setButtonText(req, res, next) {
+    req.form.options.buttonText = 'actions::continue'
```

Comment:
Could this be set within the wizard options instead?

Reply:
I think I see what actually. Maybe we need a separate action actually - `Update` or `Save` instead of `Continue`? Because there is no next step to "continue" to in these instances.

Reply:
Yep - was going to ask Andy about that

Reply:
(But, yes, it could be set in the wizard options, now you mention it)